tests/_base.js

```diff
@@ -85,49 +92,73 @@ exports.mocha = function (_module, options) {
                     rpcport: 58545,
                 },
             });
-
+            console.log(this.geth);
             yield this.geth.start();
 
             this.web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:58545'));
```

Comment:
The port is chosen because of a custom geth one?
Why using an HTTP provider here?